scripts/install_postgresql.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -e
+
+_cmd() {
+    echo "$@"
+    "$@"
+}
+
+if [[ -z ${POSTGRESQL_VERSION} ]]; then
+	echo "POSTGRESQL_VERSION not set, skipping"
+	exit 0
```

Comment:
True, this needs to be checked in a single place. I'll change it.

This is not supposed to be called directly ever, or only in very rare circumstances if you want to bypass a lot of ther `src-d/ci` logic, but it is not expected...

Reply:
I removed the check on `install_services.sh`, so it is checked in a single place.
We do not provide a default `POSTGRESQL_VERSION` because the default is empty (skipping PostgreSQL install).